pulse_actions/handlers/treeherder_resultset.py

```diff
@@ -0,0 +1,32 @@
+from mozci.mozci import trigger_missing_jobs_for_revision
+from thclient import TreeherderClient
+from pulse_actions.publisher import MessageHandler
+
+
```

Comment:
I think we need to set MEMORY_SAVING_MODE to True somewhere so it can run on Heroku.